cpp-client/cpp-client-Dockerfile

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
We shouldn't need this file anymore, right?

Reply:
See the last point in the description of the PR; I decided to leave it as a convenience for users.
I can make that more explicit in the README or in comments if you'd lile.

Reply:
Nevermind, I am removing it too.